cime_config/namelist_definition_cice.xml

```diff
@@ -1251,6 +1274,8 @@
     </desc>
     <values>
       <value>.false.</value>
+      <value run_type="branch">.true.</value>
```

Comment:
I am really confused as to why this is needed for NorESM and is not needed for CESM. This applies to all the changes where you have set 
```
<value run_type="branch">.true.</value>
<value run_type="hybrid">.true.</value>
```
Can you please point me to a failed test (e.g. ERI) where not having these changes causes problems.